artemis-core-client/src/main/resources/activemq-version.properties

```diff
@@ -20,4 +20,4 @@ activemq.version.minorVersion=${activemq.version.minorVersion}
 activemq.version.microVersion=${activemq.version.microVersion}
 activemq.version.incrementingVersion=${activemq.version.incrementingVersion}
 activemq.version.versionTag=${activemq.version.versionTag}
-activemq.version.compatibleVersionList=121,122,123,124,125,126,127,128,129
```

Comment:
@michaelandrepearce / @jbertram This change here makes it technically a 2.7.0 feature only.

Later on if we make other breaking changes into master.. it will be almost impossible to solve compatibility between 2.6.x and master.

Are you guys ok with that, or you would look into a way to make a 2.6.x change?

Reply:
sorry for me being inconsistent here... cherry-pick into 2.6.x was a piece of cake. no issues.


this is a fix anyway...  so.. it will be ok as long as the artemis-version.properties is in sync with 2.7.0

Reply:
Whats 2.7 timeline

Reply:
@michaelandrepearce I would say end of the year.. i wanted to fix large messages in AMQP first

that doesn't prevent you from cherry-picking the fix on another branch. I know you use Red Hat's downstream version.. this could be cherry-picked.